kola/tests/misc/selinux.go

```diff
@@ -56,13 +72,90 @@ func SelinuxEnforce(c cluster.TestCluster) {
 	if err != nil {
 		c.Fatalf("failed to reboot machine: %v", err)
 	}
+}
+
+// SelinuxEnforce checks that some basic things work after `setenforce 1`
+func SelinuxEnforce(c cluster.TestCluster) {
+	cmdList := []cmdCheckOutput{
+		{"/usr/sbin/getenforce", true, "Permissive"},
```

Comment:
Would it be worthwhile to track down the `$PATH` issue on RHCOS so we don't have to use the entire path for binaries in `/usr/sbin`?

Reply:
Agreed, I'll keep digging into that over in openshift/os#191.  We should be able to leave this as is until that gets sorted out, right?

Reply:
I personally think it's fine to carry it like this temporarily; maybe we should just either add a comment (either in code or in the openshift/os issue) to go back and drop them after it's resolved.

Reply:
Smart.  I'll add that in.